### ISSUE TITLE
`python-dateutil==2.8.0` version

### ISSUE THREAD

**Issue by emlazzarin** [blockchain-etl/ethereum-etl#310]
https://github.com/blockchain-etl/ethereum-etl/blob/0beebb139d8cd25134859ebec508727eb331ce74/setup.py#L39

Is this library actually version-locked against `python-dateutil` version `2.8.0`?

When installing `pandas` and `ethereum-etl`, `poetry` detects a unresolvable version conflict thanks to the version-lock.

**Comment by emlazzarin** [comment]
All tests pass with the following environment:

```
[tool.poetry.dependencies]
python = "^3.9"
web3 = "4.7.2"
eth-utils = "1.10.0"
eth-abi = "1.3.0"
python-dateutil = ">=2.8.0"
click = "7.1.2"
ethereum-dasm = "0.1.4"
base58 = "^2.1.1"
requests = "^2.27.1"
pytest = "^7.0.0"
```

Test result in `39 passed, 19 skipped, 41 warnings in 3.25s`.

**Comment by emlazzarin** [comment]
This was fixed in #311.
